commands/Server.go

```diff
@@ -0,0 +1,51 @@
+package commands
+
+import (
+  "geoffrey/endpoints"
+
+  "log"
+  "fmt"
+  "net/http"
+  "os"
+)
+
+type ServerCommand struct{}
+
+func (*ServerCommand) Name() string { return "server" }
+func (*ServerCommand) Usage() string { return "server - Start the Geoffrey server on port $PORT"}
+
+func (*ServerCommand) Execute(args []string) int {
+	addr, err := determineListenAddress()
+	if err != nil {
+		log.Fatal(err)

```

Comment:
oh you are logging.. im still reading but maybe revisit when you print vs log and see if the uses are consistent